modules/core/test/test_arithm.cpp

```diff
@@ -1563,6 +1563,21 @@ TEST(Core_round, CvRound)
     ASSERT_EQ(-4, cvRound(-3.5));
 }
 
+int naiveRound(double value)
+{
+    return (int)lrint(value);
```

Comment:
@ilya-lavrenov, it seems that Visual Studio versions prior to 2013 have no `lrint` function implemented ([blog post](http://blogs.msdn.com/b/vcblog/archive/2013/07/19/c99-library-support-in-visual-studio-2013.aspx)).